include/powsybl/iidm/Generator.hpp

```diff
@@ -21,7 +23,7 @@ class Terminal;
 class Generator : public Injection, public ReactiveLimitsHolder {
 public:
     Generator(VariantManagerHolder& network, const std::string& id, const std::string& name, const EnergySource& energySource,
-        double minP, double maxP, bool voltageRegulatorOn, const stdcxx::Reference<Terminal>& regulatingTerminal,
+        double minP, double maxP, bool voltageRegulatorOn, Terminal& regulatingTerminal,
```

Comment:
I think this should be declared as const. We miss it in the previous PR concerning the SVC